conf/pika.conf

```diff
@@ -1,141 +1,278 @@
-# Pika port
+###########################
+# Pika configuration file #
+###########################
+
+# Pika port, the default value is 9221.
+# If it's still valid, +100/+200 magic offsets could be applied.
```

Comment:
oh I didn't mean that....[facepalm] What I meant was that I can't remember whether the +1000/+2000 magic is still valid in the current version and if so, could you please add some description to it...

looking into include/pika_define.h, there are currently
`const int kPortShiftRSync      = 1000;`
`const int kPortShiftReplServer = 2000;`
in it, which means 10221 and 11221 will also be used in some circumstances(RTFC) if ``port`` is set to 9221 and IMO we'd better tell the user about it here.

Reply:
Sorry for the misunderstanding and thank you for the detailed explaination, I'll rewrite it.

Reply:
Can i express it in this way? is it  right? @fancy-rabbit 

[NOTICE] Port Magic offsets of +1000 / +2000 are existed in Pika at present,
By default, port 10221 is used for Rsync, and port 11221 is used for Replication.